netutils/constants.py

```diff
@@ -380,3 +380,8 @@
     340282366920938312347647155603121373184,
     340282366920919120650260773364972912640,
 }
+
+# End of Text characters for banner
+ETX_HEX = "\x03"
```

Comment:
I am not sure if it should be that, or the below. @jmcgill298 do you know? 
```
ETX_HEX = chr(3)
```

Reply:
@itdependsnetworks @jmcgill298 - Looks to be the same?
```
>>> chr(3)
'\x03'
>>> chr(3) == "\x03"
True
```